assets/js/buttons-examples.js

```diff
@@ -434,6 +434,23 @@ $(document).ready(function () {
             },
           ],
         },
+        {
+          type: "neu-btn",
+          label: "neumorphic",
```

Comment:
Place it in the correct alphabetical order under `special` and capitalize the first letter (same for the light and dark)